src/main/java/mat/client/cqlworkspace/InsertAttributeBuilderDialogBox.java

```diff
@@ -52,6 +54,9 @@
 import mat.model.clause.QDSAttributes;
 
 public class InsertAttributeBuilderDialogBox {
+
+    private static final Logger logger = Logger.getLogger("MAT");

```

Comment:
Should this be `Logger.getLogger(InsertAttributeBuilderDialogBox.class)` to display this class name in logged message pattern?

Reply:
It's a JUL logger (you cannot pass a class like that, only a name) and it's on a UI side.
What I found that GWT best practice is to use a root logger Logger.getLogger("") or a logger with a module name Logger.getLogger("MAT");